testing/testcafe/model/scheduler/appointment/popup.ts

```diff
@@ -13,6 +13,24 @@ export const CLASS = {
   fullScreen: 'dx-popup-fullscreen',
 };
 
+interface ISchedulerPopupExpr {
+  subject?: string;
+  startDate?: string;
+  startDateTimeZone?: string;
+  endDate?: string;
+  endDateTimeZone?: string;
+  description?: string;
+}
+
+const defaultExpr = {
```

Comment:
I think that `defaultExpr` should implement the `ISchedulerPopupExpr` interface.